src/components/ebay-badge/index.js

```diff
@@ -2,9 +2,9 @@ const processHtmlAttributes = require('../../common/html-attributes');
 const template = require('./template.marko');
 
 function getInitialState(input) {
-    const number = Number(input.number);
+    const number = Math.round(input.number);
     const imgRole = (input.type !== 'menu' && input.type !== 'icon');
-    const showBadge = !isNaN(number);
+    const showBadge = number >= 0;
```

Comment:
I just noticed this, don't we want to hide it when `number` is `0`?

Reply:
Ah.. you are right. fixing it with `number > 0`

Reply:
Good catch!